db/seeds.rb

```diff
@@ -91,27 +91,24 @@
                 start_time: 24.hours.from_now, end_time: 27.hours.from_now,
                 is_template: false, departments: [cert]
                 }])
-comms = ItemCategory.find_or_create_by_name([
-  {name: "Communication",
-   status: "Active",
-   description: "Radios, antennae, telephones, repeaters and all assorted peripherals including microphones and earpieces."
-  }
-  ])
 
-ht = ItemType.find_or_create_by_name([
-  {item_category: comms,
-   name: "Radio, Portable",
-   status: "Active",
-   description: "Handheld radios"
-  }
-  ])
+comms = ItemCategory.find_or_create_by(name: "Communication") do |item|
+          item.status = "Active",
```

Comment:
Could you please change the variable name from `item` to something else, perhaps `item_cat` ? As people try to figure out the item hierarchy, this could lead to confusion. Thanks !